src/nginx/azext_nginx/azext_metadata.json

```diff
@@ -0,0 +1,4 @@
+{
+    "azext.isExperimental": true,
```

Comment:
Do you want to release this extension in experimental, preview or GA?

Reply:
GA. Do I change this to false?